src/Steam.package/STApplicationRootComponent.class/instance/createTables.st

```diff
@@ -0,0 +1,8 @@
+setting up
+createTables
+	| session |
```

Comment:
Hmm, this method should actually exist in the store, not in the application. Two things indicate that:

1. This piece of code shows that this method is only valid for PGSql
```
(store class = STPostgresSQLStore) ifFalse: [ ^ nil ]
```
2. there are no self sends. This is the `method envy` problem we were talking about before.

Reply:
but isn't easier for the user to have this method ? 
otherwise he'll need to go in the STExampleApp find the store and then create the DB

Reply:
Not really:
 - finding the store is as simple as doing: `app store`... So it really does not add any complexity
 - it is not modular: if I add a new kind of store, should I add methods to the app too?
 - What happens if tomorrow I implement a new database (Mysql?)

Reply:
ok I see

Reply:
A general rule to follow is: put the behaviour close to the data.
Following this rule, you will get simpler designs.

Of course, as any rule we can break it, but with a good justification behind.